src/test/java/org/sasanlabs/service/vulnerability/commandInjection/CommandInjectionTest.java

```diff
@@ -0,0 +1,76 @@
+package org.sasanlabs.service.vulnerability.commandInjection;
+
+import java.io.IOException;
+import java.net.URI;
+import org.junit.jupiter.api.Test;
+import org.sasanlabs.service.exception.ServiceApplicationException;
+import org.sasanlabs.service.vulnerability.bean.GenericVulnerabilityResponseBean;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.RequestEntity;
+import org.springframework.http.ResponseEntity;
+import org.springframework.util.Assert;
+
+public class CommandInjectionTest {
+
+    CommandInjection commandInjection = new CommandInjection();
+    private String wrongUrl = "localhsost";
+
+    @Test
+    public void getVulnerablePayloadLevel1_Localhost_ExpectPingSuccess() throws IOException {
+        ResponseEntity<GenericVulnerabilityResponseBean<String>> responseEntity =
+                commandInjection.getVulnerablePayloadLevel1("127.0.0.1");
+        Assert.isTrue(
```

Comment:
We should try to use AssertJ assertions , they provide a fluent API for checks
Refer https://assertj.github.io/doc/

Reply:
Assert.equals can also be used there.

Reply:
That's a great idea, assertJ did provides better checks